source/fundamentals/crud/write-operations/insert.txt

```diff
@@ -7,6 +7,64 @@ Insert a Document
 Overview
 --------
 
-In this section, we show you how to call the write operations to **insert**
-documents from a collection in your MongoDB database.
+In this guide, you can learn how to insert documents into your MongoDB
+database. To insert a document, use insert operations.
 
+Insert operations consist of the ``insertOne()`` and ``insertMany()``
+methods. For both methods, the driver automatically generates a unique
+``_id`` for documents unless specified. 
+
+Insert Operations
+-----------------
+
+insertOne()
+~~~~~~~~~~~
+
+Use the ``insertOne()`` method to insert a single document. This method
+returns an ``InsertOneResult`` which tells us the ``insertedId`` of the
```

Comment:
I actually used ``insertedId`` as the literal term of what is within the result returned. For a single document, there's no index, but for multiple documents, there is.  And within each index, it stores the ``_id`` value as "value". Since I'm using this here as it's literal term, I think it should remain as ``insertedId``

Reply:
Is `insertedID` a class? If so could I be sent a link to its java doc page or its definition in the driver source code? 

After reviewing the [InsertOneResult](https://github.com/mongodb/mongo-java-driver/blob/bb92fd238d060d1d9d675a1c436bfb4ad38cbbc0/driver-core/src/main/com/mongodb/client/result/InsertOneResult.java#L53) source code, I believe that `insertedId` is not a class, but rather a variable representing an `_id` referenced by the `getInsertedId()` method. Because of this, I believe that saying the "`insertedId` of a document" is incorrect, as `insertedId` is a variable holding a value representing the `_id` of an inserted document within the `InsertOneResult` class.  Because `insertedID` is a variable (outside of the [`InsertOneResult`](http://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-core/com/mongodb/client/result/InsertOneResult.html?is-external=true#getInsertedId()) API as defined in the Java Doc), I believe the correct term is "id".

If you disagree with this assessment, could you please provide links to the Driver API documentation or Driver source code that provide evidence to support the stance that `insertedID` is the correct term?